config.toml

```
@@ -188,7 +187,7 @@ language_alternatives = ["en"]
 [languages.it]
 title = "Kubernetes"
 description = "Production-Grade Container Orchestration"
```

nit: this text isn't translated yet.
```suggestion
description = "Orchestrazione di Container in produzione"
```